app/controllers/HomePageController.scala

```diff
@@ -35,6 +35,8 @@ class HomePageController @Inject() (
     with I18nSupport {
 
   def onPageLoad: Action[AnyContent] = (identify andThen profileAuth andThen getOrCreate) { implicit request =>
-    Ok(view())
+    // TODO: Actually check whether the file is ready or not by calling something
+    val isDownloadReady = false
```

Comment:
Nice, so we'll be dealing with how long we are gonna display the banner (I believe 30 days from when file is ready) in a later ticket or a spike?